tools/db_crashtest.py

```diff
@@ -122,6 +122,15 @@ def is_direct_io_supported(dbname):
 
 whitebox_simple_default_params = {}
 
+atomic_flush_params = {
```

Comment:
One thing that's difficult about db_stress is knowing whether a params map overrides values in another params map. I never came up with a good solution but tried to document it on lines 12-17 - do you mind updating those comments to reflect this new map?